examples/samples/sessionState.ts

```diff
@@ -0,0 +1,146 @@
+import { generateUuid, SendableMessageInfo, ReceiveMode, Namespace } from "../../lib";
+import * as dotenv from "dotenv";
+dotenv.config();
+
+const connectionString = process.env.SERVICEBUS_CONNECTION_STRING || "";
+const userEventsQueue = process.env.QUEUE_NAME || "";
+
+console.log("connection string: ", connectionString);
+console.log("path: ", userEventsQueue);
+
+let ns: Namespace;
+
+/*
+  This sample demonstrates usage of SessionState.
+  Ref - https://docs.microsoft.com/en-us/azure/service-bus-messaging/message-sessions
+
+  We take for example the context of an online shopping app and see how we can use Session State
+  to implement the maintaining of shopping cart information completely on server side i.e.,
+  remembering the user's shopped items even if they leave the site and return later.
+
+  The scenario in sample walks through user activity of two customers Alice and Bob.
+  Alice shops for 3 items and checks out, whereas Bob adds 3 items and leaves without
```

Comment:
"Bob adds 3 items and" should either be "Bob shops for 3 items and". Otherwise it looks like Alice and Bob did 2 different activities.

Or since you later refer to items in cart, may be both Alice and Bob should "add 3 items and. .."